### ISSUE TITLE
Document distribution guarantees

### ISSUE THREAD

**Issue by SevereOverfl0w** [jimpil/duratom#17]
Hi, not an actual user, so don't prioritise this.

It occurred to me when browsing though, that it's not clear to me whether this is suitable for use from multiple servers simultaneously or not. For example if host A and B both call `(swap! a inc)` simultaneously, is it guaranteed that the number will increase by 2, or will it potentially increase only by 1?

**Comment by jimpil** [comment]
Hi there,

Let me make sure I understand the use-case first... You are thinking of a situation where two programs (running on different machines) initialise a `duratom` each against the same (remote) storage backend (e.g. PostgresDB), right? You obviously can't share an atom (nor a duratom) between different processes, so that is the only interpretation I can think of that makes sense...

If so, the answer is it depends on the backend, but for  example in the Postgres case, both commits  would succeed. That said, `duratom` synchronises with the backend only once - upon initialisation. After initialisation, commits flow one way - from the duratom to the storage backend *only*. This means that your duratoms on the various hosts are bound to become out-of-sync eventually. 

TL;DR;
The state of a backend should only be affected by the duratom wrapping that backend. `duratom` provides no distributed semantics whatsoever. 

Hope that helps...

**Comment by SevereOverfl0w** [comment]
That makes perfect sense, yep! Thanks.
